CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.13.3] - 2024-01-11
+### Changed
+- Type hints `Sequence[str]` are now replaced with `SequenceNotStr[str]`.
+
```

Comment:
We doing this now? 🤔

Reply:
Thinking is to have released version consistent with the docs. However, you are right we do not do this, (and we release so often that it will not be outdated for long anyways). Removing as it is no change to functionality.